### ISSUE TITLE
add CI

### ISSUE THREAD

**Issue by sinarueeger** [sinarueeger/ggGWAS#32]
like in 
https://www.hvitfeldt.me/blog/usethis-workflow-for-package-development/

**Comment by sinarueeger** [comment]
- `usethis::use_travis()
- `usethis::use_coverage(type = c("codecov"))`

**Comment by sinarueeger** [comment]
problem with `ggrastr`, not `available (for R version 3.5.3)`

**Comment by sinarueeger** [comment]
`ggrastr` works now bc this snippet is added in [DESCRIPTION](https://github.com/sinarueeger/ggGWAS/blob/master/DESCRIPTION):

```
Remotes: 
    sinarueeger/GWAS.utils,
    VPetukhov/ggrastr
```
